tests/MaintenanceLockTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Symbiote\QueuedJobs\Tests;
+
+use SilverStripe\Control\Director;
+use SilverStripe\Core\Config\Config;
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+
+/**
+ * Class MaintenanceLockTest
+ *
+ * @package Symbiote\QueuedJobs\Tests
+ */
+class MaintenanceLockTest extends AbstractTest
+{
+    /**
+     * @param $lockFileEnabled
+     * @param $fileExists
+     * @param $lockActive
+     * @dataProvider getMaintenanceCases
+     */
+    public function testEnableMaintenanceIfActive($lockFileEnabled, $fileExists, $lockActive)
+    {
+        Config::nest();
```

Comment:
Config nesting is done implicitly by SapphireTest in SS4 - you don't need this =)